### ISSUE TITLE
Automate tab tracker

### ISSUE THREAD

**Issue by jin** [chromelens/chromelens#15]
Instead of pressing tab manually, automate tabbing to quickly generate the tab flow graph of a webpage with a single click.

**Comment by jmuheim** [comment]
Very good idea. In fact, I thought the feature was broken because I didn't know that I have to tab manually.

**Comment by jmuheim** [comment]
Also, when holding the tab key tabbing seems to be too fast, and the red line isn't drawn.

**Comment by jin** [comment]
@jmuheim Yep, currently the lines are drawn only on the tab keyup event: https://github.com/chromelens/chromelens/blob/master/tracer/trace_tab_path.js#L32

**Comment by jmuheim** [comment]
Interesting. Is there a specific reason for this?

**Comment by jin** [comment]
@jmuheim No specific reason, just the way it was done from the beginning.
